warehouse/accounts/views.py

```diff
@@ -874,7 +874,7 @@ def _error(message):
         ),
         queue="success",
     )
-    return HTTPSeeOther(request.route_path("manage.account"))
+    return HTTPSeeOther(request.route_path("manage.account.two-factor"))
```

Comment:
Hmmm, should this be wrapped in a conditional? If I already have 2FA and add/verify a new address I’ll still be thrown to the 2FA page.

Reply:
Nice catch, will handle that behavior.

Reply:
Added!